spreadsheet/row.go

```diff
@@ -98,7 +98,24 @@ func (r Row) AddCell() Cell {
 // to the slice will have no effect.
 func (r Row) Cells() []Cell {
 	ret := []Cell{}
+	lastIndex := -1
 	for _, c := range r.x.C {
+		if c.RAttr == nil {
+			fmt.Println("RAttr is nil for a cell, skipping.")
+			continue
+		}
+		ref, err := reference.ParseCellReference(*c.RAttr)
+		if err != nil {
+			fmt.Println("RAttr is incorrect for a cell: " + *c.RAttr + ", skipping.")
```

Comment:
Please use debug logging, not directly to stdout.  Don't we have a logger in the package?
should be like log.Debugf()...

Reply:
There is kind of logger in the package, I fixed Cells() to use it, but so far it anyway just prints to Stdout